google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -20,6 +20,9 @@ Code inspections for AppEngine Java code.</description>
     <component>
       <implementation-class>com.google.gct.idea.CloudToolsPluginInitializationComponent</implementation-class>
     </component>
+    <component>
+      <implementation-class>com.google.gct.idea.debugger.CloudDebugProcessStateCollector</implementation-class>
```

Comment:
@loosebazooka 
The main property of ApplicationComponents is that they hook into the IntelliJ defined plugin lifecycle. 

If you just want a plugin scoped object you should use a service.